### ISSUE TITLE
Commander CLI distributed through an internal nexus

### ISSUE THREAD

**Issue by mkiddTempus** [tj/commander.js#1682]
Scenario:

I want someone internal to Acme to execute `npm install -g @Acme/cli` and the Commander cli I produced that works with a `cli doSomething` will also work everywhere else it has been installed due to the internal Acme company nexus. 

I feel a little naive right now as this feels like a doable thing but I didn't see anything in the documents and that may mean it's more of a Javascript/Typescript general question.

**Comment by shadowspawn** [comment]
Yes, general question about how to write a CLI in an npm package.

A quick overview is you list your `cli` in the `bin` section in your `package.json`. When the package is installed globally, it adds a link to the binary in (roughly speaking) the same folder as the `npm` binary. 

An example with my `forest-arborist` package which adds a `fab` command. Notice the first line after the install where it shows the link it has added:
```
% command -v npm
/usr/local/bin/npm
% npm install -g @shadowspawn/forest-arborist
/usr/local/bin/fab -> /usr/local/lib/node_modules/@shadowspawn/forest-arborist/dist/src/cli.js
+ @shadowspawn/forest-arborist@8.1.2
removed 1 package and updated 7 packages in 6.552s
% command -v fab
/usr/local/bin/fab
```
 
Relevant line in `package.json`: https://github.com/shadowspawn/forest-arborist/blob/1b3529a6a23a4ba33ca3f224d1dfbd7b658d8629/package.json#L10
